### PR TITLE
Improve devtools launching

### PR DESCRIPTION
**Pull request by devoncarew**
Improve devtools launching:

- no longer route devtools starting up output to the message console (fixes an issue where we steal focus from the run/debug console)
- address an issue where some hyperlinks in the performance tool window would launch devtools, but would not perform the devtools app install flow (would fail if devtools was not already installed)
- remove the deprecated `WebDevManager.java` class

This is in prep for additional devtools launching improvements (https://github.com/flutter/flutter-intellij/issues/3503).

### CODE REVIEW

File: src/io/flutter/devtools/DevToolsManager.java [hunk 1]
```diff
@@ -194,21 +195,28 @@ private boolean isBazel(Project project) {
...
+          devToolsInstance = instance;
+
+          devToolsInstance.openBrowserAndConnect(uri, page);
+        }, () -> {
+          // Listen for closing; null out the devToolsInstance.
+          devToolsInstance = null;
+        });
+      }
+      else {
+        devToolsInstance = null;
```
Comment (kenzieschmoll) [inline src/io/flutter/devtools/DevToolsManager.java]: If we hit this case, is there any feedback we need to show to the user?

File: src/io/flutter/view/PerfFPSTab.java [hunk 2]
```diff
@@ -95,11 +96,16 @@ private void buildUI() {
...
   private void openInDevTools() {
+    final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
+
     // open the timeline view
-    DevToolsManager.getInstance(app.getProject()).openBrowserAndConnect(
-      app.getConnector().getBrowserUrl(),
-      "timeline"
-    );
+    if (devToolsManager.hasInstalledDevTools()) {
+      devToolsManager.openBrowserAndConnect(app.getConnector().getBrowserUrl(), "timeline");
```
Comment (kenzieschmoll) [inline src/io/flutter/view/PerfFPSTab.java]: perhaps put this in a method inside `DevToolsManager` to consolidate the two uses here in the memory tab below. Something like:
```
public void openToScreen(String screen) {
  openBrowserAndConnect(app.getConnector().getBrowserUrl(), screen);
}
```

or even better consolidate all the conditional logic in DevToolsManager, and just call `devToolsManager.openToScreen("timeline")` here (and below for memory)
```
public void openDevTools(String screen) {
  if (hasInstalledDevTools()) {
     openToScreen(screen);
  } 
  else {
      final CompletableFuture<Boolean> result = installDevTools();
      result.thenAccept(o -> openToScreen(screen));
  }
}

private void openToScreen(String screen) {
  openBrowserAndConnect(app.getConnector().getBrowserUrl(), screen);
}
```

File: src/io/flutter/devtools/DevToolsManager.java [hunk 3]
```diff
@@ -133,8 +132,26 @@ public void openBrowserAndConnect(String uri) {
...
+    openBrowserImpl(uri, screen);
+  }
+
+  /**
+   * Open a browser connected to DevTools with the given (optional) screen.
+   * <p>
+   * Calling this method may cause DevTools to be installed (in which case, this API will provide
+   * appropriate progress feedback to the user).
+   */
+  public void openToScreen(@NotNull FlutterApp app, @Nullable String screen) {
```
Comment (kenzieschmoll) [inline src/io/flutter/devtools/DevToolsManager.java]: As long as this behaves properly when screen is null, lgtm

Comment (devoncarew) [reply src/io/flutter/devtools/DevToolsManager.java]: Yup, it should (it's existing logic in DevToolsUtils.generateDevToolsUrl()).

**Pull request merged**